backend/tests/test_server.py

```diff
@@ -0,0 +1,87 @@
+import json
+from unittest import mock
+
+import pytest
+
+from ..redis_schema import CurrentUsers, OwnsConnection, Users
+from .app_fixtures import *
+from .user_fixtures import *
```

Comment:
I think I'm just a curmudgeon but it makes it a little harder to read if you do this as opposed to `import .app_fixtures as app_fixtures` (is this a thing you can do?) because without that, it's a little harder to see where the fixtures and stuff are coming from.

Reply:
Yea thats just gonna have to be too bad